migration/src/m0000331_create_status.rs

```diff
@@ -0,0 +1,95 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .create_table(
+                Table::create()
+                    .table(Status::Table)
+                    .col(
+                        ColumnDef::new(Status::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment(),
+                    )
+                    .col(ColumnDef::new(Status::Slug).string().not_null())
+                    .col(ColumnDef::new(Status::Name).string().not_null())
+                    .col(ColumnDef::new(Status::Description).string())
+                    .primary_key(Index::create().col(Status::Id))
+                    .to_owned(),
+            )
+            .await?;
+
+        let db = manager.get_connection();
+
+        insert(db, "affected", "Affected", "Vulnerabililty affects").await?;
+        insert(
+            db,
+            "not_affected",
+            "Not Affected",
+            "Vulnerabililty does not affect",
+        )
+        .await?;
+        insert(db, "fixed", "Fixed", "Vulnerabililty is fixed").await?;
+        insert(
+            db,
+            "under_investigation",
+            "Under Investigation",
+            "Vulnerabililty is under investigation",
+        )
+        .await?;
+        insert(db, "fixed", "Fixed", "Vulnerabililty is fixed").await?;
+        insert(
+            db,
+            "recommended",
+            "Recommended",
+            "Vulnerabililty is fixed & recommended",
+        )
+        .await?;
+
```

Comment:
Why not just a rusty enum for these? Will we need to add more later without publishing a new release?

Reply:
While there are some intrinsic statuses, I don't deign to know all statuses customers might want to apply, in the future. So leaving the architecture open to extension at runtime.

I waffle about using the slug as the actual PK.